modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc

```diff
@@ -5920,6 +5920,132 @@ Total database accesses: 5, total allocated memory: 1488
+| +CartesianProduct           |  3 |                                                                                                  |              0 |    1 |       0 |           1864 |                        |     0.847 | In Pipeline 2       |
+| |\                          +----+--------------------------------------------------------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+---------------------+
+| | +Filter                   |  4 | stefan.name = $autostring_1                                                                      |              1 |    1 |      28 |                |                        |           |                     |
+| | |                         +----+--------------------------------------------------------------------------------------------------+----------------+------+---------+----------------+                        |           |                     |
+| | +NodeByLabelScan          |  5 | stefan:Person                                                                                    |             14 |   14 |      15 |            392 |                    1/0 |     0.482 | Fused in Pipeline 1 |
+| |                           +----+--------------------------------------------------------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+---------------------+
+| +Filter                     |  6 | chris.name = $autostring_0                                                                       |              1 |    1 |      28 |                |                        |           |                     |
+| |                           +----+--------------------------------------------------------------------------------------------------+----------------+------+---------+----------------+                        |           |                     |
+| +NodeByLabelScan            |  7 | chris:Person                                                                                     |             14 |   14 |      15 |            376 |                    3/0 |     2.092 | Fused in Pipeline 0 |
++-----------------------------+----+--------------------------------------------------------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+---------------------+
+
+Total database accesses: 125, total allocated memory: 24349
+
+----
+
+======
+
+
+[[query-plan-stateful-shortest-path-all]]
+== StatefulShortestPath(All)
+// StatefulShortestPath(All)
+
+The `StatefulShortestPath(All)` operator finds shortest paths from a previously matched node variable to an endpoint that was not previously matched.
+It uses a breath first search algorithm.
+
+
+.StatefulShortestPath(All)
+======
+
+.Query
+[source, cypher]
+----
+PROFILE
+MATCH (chris {name: 'Chris'})
```

Comment:
You probably want a `Person` label here?

Reply:
Yes, good catch. This, in combination with the index on `Person.name`, solved my issue in https://trello.com/c/eQvCFdzF/2587-optimization-opportunity-for-statefulshortestpath